app/views/hyrax/base/_work_title.erb

```diff
@@ -1,9 +1,16 @@
 <% presenter.title.each_with_index do |title, index| %>
-    <% if index == 0 %>
-        <h1>
-            <%= title %> <%= presenter.permission_badge %> <%= presenter.workflow.badge %>
-        </h1>
-    <% else %>
-        <h1><%= title %></h1>
-    <% end %>
+  <div class="row">
+    <div class="col-sm-8">
+      <% if index == 0 %>
+        <h2><%= title %>
```

Comment:
Do we have an `h1` on this page?  Not doing so would be an accessibility concern I believe.

Reply:
We do, it's higher on the page in the _work_type partial.